### DISCUSSION TITLE
The uploaded video, click download shows 502 error

### DISCUSSION THREAD

**Discussion by LaoJiuYes** [requarks/wiki#5488] [Error / Bug Report] [answered]
The uploaded video, click download shows 502 error, I use nginx proxy, I tried to change the download limit, but it did not work, I do not know what happened

**Comment by NGPixel** [comment, marked as answer]
You shouldn't store videos (or any large files) in Wiki.js directly. Your server is likely timing-out trying to serve the file, causing a 502 error in your reverse-proxy.

Host large files on a proper CDN or object store service.

**Reply by LaoJiuYes** [reply]
Okay, I understand, thank you for the answer
